Apromore-Core-Components/Apromore-Manager/src/main/java/org/apromore/security/impl/UsernamePasswordAuthenticationProvider.java

```diff
@@ -222,7 +223,7 @@ private Authentication authenticatedToken(User user, Authentication original) {
     }
 
     /**
-     * Retrieves the correct ROLE type depending on the access level, where access level is an Integer.
+     * Retrieves the correct ROLE or PERMISSION type depending on the access level, where access level is an Integer.
      * Basically, this interprets the access value whether it's for a regular user or admin.
      *
      * @param access an integer value representing the access of the user
```

Comment:
This doc comment seems to be out of date, since the `access` parameter is a set of `Role`s rather than an integer.